Python/Product/Debugger/DebugAdapterProcess.cs

```diff
@@ -178,6 +179,24 @@ sealed class DebugAdapterProcess :
             if (_stream != null) {
                 _stream.Dispose();
             }
+
+            if (_process.ExitCode == 126) {
```

Comment:
What if debugging started successfully but the user's program exited with code 126? Can you do this only if debuggee didn't connect back?

Reply:
sure i can restrict it.